src/components/DropDownList/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
+import './index.css'
+function DropDownList({ items, setItem, placeholder }) {
+    const [selected,setSelected] = useState('');
+    const options = items.map((item) =>
+        <div 
```

Comment:
Wouldn't it better to have Option as a separate (private) component?

Reply:
I agree! will do that in the next version